packages/dart/sshnoports/README.md

```diff
@@ -4,13 +4,12 @@
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports/badge)](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports)
 [![SLSA 3](https://slsa.dev/images/gh-badge-level3.svg)](https://slsa.dev)
 
-# SSH! No ports
+# NoPorts Cli Binaries
```

Comment:
s/Cli/CLI/

Reply:
fixed in [0e23e54](https://github.com/atsign-foundation/noports/pull/1613/commits/0e23e54562176433076875067b0a3f7c94563856)